README.md

```diff
@@ -52,74 +52,93 @@ const pow = createPow({ host })
 
 ```typescript
-const { token } = await pow.ffs.create() // save this token for later use!
+import { POW } from "@textile/powergate-client"
+
+async function getToken (pow: POW) {
+  const { token } = await pow.ffs.create() // save this token for later use!
+  return token
+}
 ```
 
 Currently, the returned auth token is the only thing that gives you access to your FFS instance at a later time, so be sure to save it securely.
 
 Once you have an auth token, either by creating a new FFS instance or by reading one you previously saved, set the auth token you'd like the Powergate client to use.
 
 ```typescript
+import { createPow } from "@textile/powergate-client"
+
+const host = "http://0.0.0.0:6002" // or whatever powergate instance you want
+
+const pow = createPow({ host })
+
+const authToken = '<generated token>'
+
 pow.setToken(authToken)
 ```
 
 Now, the FFS API is available for you to use.
 
 ```typescript
 import fs from "fs"
-import { ffsTypes } from "@textile/powergate-client"
+import { ffsTypes, POW } from "@textile/powergate-client"
 
-// get wallet addresses associated with your FFS instance
-const { addrsList } = await pow.ffs.addrs()
+async function ffsMethods (pow: POW) {
+  // get wallet addresses associated with your FFS instance
+  const { addrsList } = await pow.ffs.addrs()
 
-// create a new address associated with your ffs instance
-const { addr } = await pow.ffs.newAddr("my new addr")
+  // create a new address associated with your ffs instance
+  const { addr } = await pow.ffs.newAddr("my new addr")
 
-// get general info about your ffs instance
-const { info } = await pow.ffs.info()
+  // get general info about your ffs instance
+  const { info } = await pow.ffs.info()
 
-// cache data in IPFS in preparation to store it using FFS
-const buffer = fs.readFileSync(`path/to/a/file`)
-const { cid } = await pow.ffs.stage(buffer)
+  // cache data in IPFS in preparation to store it using FFS
+  const buffer = fs.readFileSync(`path/to/a/file`)
+  const { cid } = await pow.ffs.stage(buffer)
 
-// store the data in FFS using the default storage configuration
-const { jobId } = await pow.ffs.pushStorageConfig(cid)
+  // store the data in FFS using the default storage configuration
+  const { jobId } = await pow.ffs.pushStorageConfig(cid)
 
-// watch the FFS job status to see the storage process progressing
-const cancel = pow.ffs.watchJobs((job) => {
-  if (job.status === ffsTypes.JobStatus.CANCELED) {
-    console.log("job canceled")
-  } else if (job.status === ffsTypes.JobStatus.FAILED) {
-    console.log("job failed")
-  } else if (job.status === ffsTypes.JobStatus.SUCCESS) {
-    console.log("job success!")
-  }
-}, jobId)
+  // watch the FFS job status to see the storage process progressing
+  const jobsCancel = pow.ffs.watchJobs((job) => {
+    if (job.status === ffsTypes.JobStatus.JOB_STATUS_CANCELED) {
```

Comment:
a handful of good catches alread

Reply:
nice, forgot about those enum name updates!